overlord/snapstate/conflict.go

```diff
@@ -76,7 +76,7 @@ func RegisterAffectedSnapsByKind(kind string, f AffectedSnapsFunc) {
 	affectedSnapsByKind[kind] = f
 }
 
-func affectedSnaps(t *state.Task) ([]string, error) {
+func AffectedSnapsByTask(t *state.Task) ([]string, error) {
```

Comment:
reading the name it feels like this should return some sort of a map of task id -> list of affected snaps. How about SnapsAffectedByTask?

Reply:
Good point, updated!